src/app/components/widgets/charts/base-chart.class.ts

```diff
@@ -1299,7 +1299,6 @@ export class BaseChartClass extends BaseWidget implements OnInit, AfterViewInit
         const yAxis = this.chartConfig.yAxis as Highcharts.YAxisOptions;
         const xAxis = this.chartConfig.xAxis as Highcharts.XAxisOptions;
 
-        yAxis.min = 0;
```

Comment:
We need to check negative values first. More details in the comment

Reply:
Done